### PR TITLE
Fix: Custom Scoreboard Reset Issue

### PR DESCRIPTION
**Pull request by j10a1n15**
## Changelog Fixes
+ Fixed Scoreboard Reset not updating the draggable list. - j10a1n15

### CODE REVIEW

**Comment by hannibal002** [comment]
whats going on here? why does the user have to restart their game?

**Comment by j10a1n15** [comment]
The draggable list becomes unresponsive after pressing the reset button
image
reopening /sh doesnt fix this

**Comment by Thunderblade73** [comment]
You need to copy that list. Not assign it. This should fix it.

File: src/main/java/at/hannibal2/skyhanni/config/features/gui/customscoreboard/CustomScoreboardConfig.java [hunk 1]
```diff
@@ -33,7 +33,7 @@ public class CustomScoreboardConfig {
     @ConfigEditorDraggableList()
     public List<ScoreboardElement> scoreboardEntries = new ArrayList<>(ScoreboardElement.defaultOption);
 
-    @ConfigOption(name = "Reset Appearance", desc = "Reset the appearance of the advanced scoreboard.")
+    @ConfigOption(name = "Reset Appearance", desc = "Reset the appearance of the advanced scoreboard.\n§cYou will need to restart your game to apply the changes.")
     @ConfigEditorButton(buttonText = "Reset")
     public Runnable reset = () -> {
         scoreboardEntries = ScoreboardElement.defaultOption;
```
Comment (Thunderblade73) [inline src/main/java/at/hannibal2/skyhanni/config/features/gui/customscoreboard/CustomScoreboardConfig.java]: Here, this assigns a the static list defaultOptions. But you need to clear the list and add in all elements from default options. This should fix it.

**Review by Thunderblade73 — APPROVED** [review]
Worked.

**Pull request merged**